### DISCUSSION TITLE
Faulty Sectors

### DISCUSSION THREAD

**Discussion by hcm609** [filecoin-project/lotus#7738] [Storage Provider] [answered]
Hi.
I updated the new release of lotus and I started paying fines (real money) due to the Faults.
I had to shutdown the miner and restart it several times for ~ 2hours so chain found me in having faults to all sectors.
See stats:
Power | 2.41 Ti / 13.9 Ei (0.0000%)
Raw | 2.406 TiB / 13.91 EiB (0.0000%)
Committed | 40.44 TiB
Proving | 40.41 TiB (38 TiB Faulty, 93.97%)
Can you please tell me what I can do to recheck and reevaluate them so as to be online again?

**Comment by SgtCoin** [comment, marked as answer]
Resolved in https://filecoinproject.slack.com/archives/CPFTWMY7N/p1638810734313900
